### ISSUE TITLE
SFM stops functioning if you cancel mid movie render

### ISSUE THREAD

**Issue by PatrickJnr** [Facepunch/sbox-issues#202]
**Describe the bug**
If at anytime you cancel a movie render, you're forced to close sbox-dev due to SFM not allowing new inputs, also asset browser and related tools don't respond to user input

**To Reproduce**

File > Export > Movie > Export Movie
Cancel at any time

**Expected behavior**
To be able to resume using SFM and the rest of the toolset

**Comment by aylaylay** [comment]
Just pushed a fix for this, please close the issue if it works for you and you're happy with it

**Comment by PatrickJnr** [comment]
Thank you so much @aylaylay it works a treat! 💖

**Comment by PatrickJnr** [comment]
![image](https://user-images.githubusercontent.com/3267523/129674350-0ebc81b8-e45a-4b36-aa88-9bc59d8bf5ca.png)

New issue, it goes to 99% but in the log at the bottom is says `finished rendering 163 frames` so it finished, but SFM thinks otherwise, but checking the video file it's there fully working, just the progress ends at 99% in the SFM Window. Pressing cancel allows me to use SFM, It's a minor bug, but that's fine.

**Comment by aylaylay** [comment]
Is this something that just started happening or has it always been like that? The change I made to fix the original issue shouldn't be affecting that.

**Comment by PatrickJnr** [comment]
This only started since the new fix.

**Comment by aylaylay** [comment]
Pushed a fix, give that a go

**Comment by PatrickJnr** [comment]
ok, now it hits 100%, but does not auto close the render window 
![image](https://user-images.githubusercontent.com/3267523/129690691-db24aa4b-7a40-45b9-b6e8-3f4aacfbdba2.png)

**Comment by aylaylay** [comment]
Hmm it was closing in my tests, I'll try again

**Comment by PatrickJnr** [comment]
Let me try again, I don't think steam pulled the update when I tested

**Comment by PatrickJnr** [comment]
Ok, it closed automatically this time! Thank you once again for fixing this!
